### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by MostafaShahat2021**
## In this PR:
- Make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).
- Make sure that I used the correct [Gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md).
 - Make sure that I documented my work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).
 - Follow Microverse's list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).
- ⭐️The project is a single-page application (SPA) built with React and Redux.
- ⭐️The data retrieved from the API is stored in the Redux store.
- ⭐️Deplay the SPA on Render platform.
- ⭐️ On the home page the app renders a list of available remote jobs.
- ⭐️User can filter the list by job title or job category.
- ⭐️The user can check the job details on the details page.
- ⭐️ [Live Demo Link](https://remotive-app-spa-with-react-redux.onrender.com/)
- ⭐️ [Project documentation video link](https://drive.google.com/file/d/1QByabRLvCzKWgbdk0HGgl9bqLmlYwo7d/view?usp=sharing)

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+  width: 50%;
+  padding: 0.5rem;
+  margin: 1rem auto;
+}
+
+.cards {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 5px;
```
Comment (VICTOR-CHUKWU) [inline src/index.css:76]: - [ ] you have done fantastic work on your designs, however, there is a minute change to make, if you look closely to the given design, you will see that there is no space or gap between cards, but yours have spaces tho tiny between cards, kindly remove it
your design :point_down: 
Screenshot from 2023-09-08 11-32-14
given design :point_down: 
Screenshot from 2023-04-28 11-04-25

File: src/index.css [hunk 2]
```diff
@@ -0,0 +1,169 @@
...
+}
+
+.details-card ul li {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 95%;
+  padding: 5px;
+  margin: 5px;
+}
```
Comment (VICTOR-CHUKWU) [inline src/index.css:141]: - [ ] kindly make your date in human-readable form and also make sure the text doesn't stick out of its container
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @MostafaShahat2021 , 
## STATUS: CHANGES REQUIRED :recycle: 
Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.css (hunk 1 above)
Comment (MostafaShahat2021) [reply src/index.css:76]: Thanks , I'll fix it 👍

File: src/index.css (hunk 2 above)
Comment (MostafaShahat2021) [reply src/index.css:141]: Thanks I'll fix it 👍

**Comment by MostafaShahat2021** [comment]
Hi @VICTOR-CHUKWU 👋, Thanks for your efforts in reviewing my code. 😄

**Review by brainconnect93 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @MostafaShahat2021, 👋 👋  

Your project is complete! Great job for following the project requirement🥇and implementing the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 




## Good Points 👍

- Descriptive Pull Request title & summary. ✔️
- Navigation with `React router` ✔️
- Followed design guidelines ✔️
- All test case passes✔️
- Good use of Git flow ✔️

## Optional changes

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brainconnect93 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MostafaShahat2021** [comment]
Hi  @brainconnect93 👋, Thanks a lot :100: for your efforts in reviewing my code :smile: , I really appreciate it 😊.

**Pull request merged**